gemini/master.py

```diff
@@ -50,6 +50,24 @@ def join() -> str:
 
     return jsonify(response)
 
+@app.route('/v1/nodes/auth', methods=['GET', 'POST'])
+def auth() -> str:
+    node_id = request.args.get('node_id')
+    password = request.get_json('password')
```

Comment:
What's the difference between how you get `node_id` and `password`? Is node ID fetched from the request query parameters, and password is from the JSON body?

Reply:
yes node_id is get request, password is json body